src/blog/announcing-flowforge-cloud.md

```diff
@@ -1,27 +1,24 @@
 ---
 title: Announcing FlowForge Cloud
 subtitle: Hosting your Node-RED, so you don't have to.
-description: Join the wait list for our cloud offering.
+description: Join the waitlist for our cloud offering.
 date: 2022-02-23 19:44:00.0
 authors: ["zeger-jan-van-de-weg"]
 ---
 
-As an open core company, anyone is open to [download and install][install-docs]
-our [latest release][release-v02]. In many cases this is a great solution, it
-allows for custom setups in your own environment. This isn't for everyone
-however, many use-cases are will not need custom firewall configuration, or want
-to maintain and monitor a bespoke solution for managing their Node-RED instances.
+As an open core company, anyone is free to [download and install][install-docs]
+our Community Edition. In many cases this is a great solution, it
```

Comment:
Have discussed the 'Community Edition' before? It implies there's an 'Enterprise edition', which also doesn't exist. Think it's better to not get into the details too much? And either it's self managed like you recommended, or FlowForge managed?

```suggestion
our platform. In many cases this is a great solution, it
```

Reply:
yes good point really the difference here is "our hosted platform" vs "self-managed" Community vs Enterprise is a different comparison (there may even be 'enterprise' hosted at some point in the future